examples/bank/bank_db/scenarios/users.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should this file not be called `users-only.json` as per the docs?

Reply:
Yes, it should be `users-only.json`